src/components/Common/Module/FieldPicker.vue

```diff
@@ -147,6 +233,22 @@ export default {
       // Remove selected fields
       return this.allFields.filter(a => !this.fields.some(f => a.name === f.name))
     },
+
+    selectAll () {
```

Comment:
selectAll indicates an action (method). This should be more like allSelected, allUnselected